spec/features/course/assessment/answer/text_response_answer_spec.rb

```diff
@@ -40,13 +41,27 @@
 
           # The file should show in the dropzone
           expect(dropzone).to have_css('span', text: 'text.txt')
-
-          click_button('Save Draft')
+          wait_for_job
 
           expect(dropzone).to have_no_css('span', text: 'text.txt')
           expect(file_view).to have_css('span', text: 'text.txt')
           expect(file_view).to have_css('span', count: 2)
           expect(answer.specific.attachments).not_to be_empty
+
+          # Attempting to delete the file that we have just uploaded
+          # Click on the delete button and confirm
+          delete_button = file_view.find('svg')
+          delete_button.click
+          click_button('Delete')
+          wait_for_job
```

Comment:
What job? got job here?

Reply:
it serves as a precaution only, since deleting file might take time and it might have risk that while the job is still on progress, it already progressed into the next line and then the line expect will fail unexpectedly (it happens, though very rarely)

Reply:
Then dont use wait_for_job right. What's the purpose of wait_for_job?

Reply:
this deleting file is actually purely backend job
Once the Delete button is clicked, then it's FE sending the API to the BE together with all the information needed for BE to do the deleting of the job, then BE get back to FE and informing that the deletion has taken place

By this definition, I think wait_for_job should be the one being used here. Please let me know if you have other thoughts.

Reply:
It is for backend “job”. Like the one when you do polling. Not backend job to return delete.

Reply:
understand more now on this wait_for_job
I have created another one called wait_for_autosave, and adjusted accordingly

Done.